### PR TITLE
core-node docs: complete node

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/_includes/core-nodes/complete-use-case.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+In Node-RED, certain nodes like WebSocket-out and MQTT-out do not have outputs to connect with. The Complete node in Node-RED can be helpful when you need to know when a process is done by those nodes.
+
+# Example
+
+1. In the example flow below, we have a WebSocket server, the Inject node sends data to the WebSocket server, and upon successful transmission to the WebSocket server, a Complete node handles the event.
+
+{% renderFlow %}
+
+[
```
Comment (ZJvandeWeg) [inline src/_includes/core-nodes/complete-use-case.md:38]: Please minimise this

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/_includes/core-nodes/complete-use-case.md (hunk 1 above)
Comment (sumitshinde-84) [reply src/_includes/core-nodes/complete-use-case.md:38]: done !

Comment (ZJvandeWeg) [reply src/_includes/core-nodes/complete-use-case.md:38]: It's not?

File: src/_includes/core-nodes/complete-use-case.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+{% endrenderFlow %}
+
+2. In this example, we simulate a time-consuming task using setTimeout in a function named "Long running process". After completion, we mark success with `node.done()` using a Complete node. Then, perform further action we process the result and assign it to `msg.payload` for further use.
+
+{% renderFlow %}
+
+[
+    {
+        "id": "f3567f52f2c9f98b",
+        "type": "function",
```
Comment (ZJvandeWeg) [inline src/_includes/core-nodes/complete-use-case.md:145]: Why not make a get request? Network is a much better example than using a function node

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/_includes/core-nodes/complete-use-case.md (hunk 2 above)
Comment (sumitshinde-84) [reply src/_includes/core-nodes/complete-use-case.md]: Okay, Zj, I will update it with a GET request.

Comment (sumitshinde-84) [reply src/_includes/core-nodes/complete-use-case.md]: done !

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**